src/frequentQuestions/style.css

```diff
@@ -19,7 +72,25 @@
   width: 70%;
 }
 
-.questionList {
-  max-height: 60vh;
-  overflow: auto;
-}
\ No newline at end of file
+.questions-footer{
+  /* border: 2px solid blueviolet; */
+
+}
+
+/**
+ * Reset button styles.
+ */
+ button {
```

Comment:
Everything looks good, except that here, I think it'd be better to use a classname for this rather than specify an element. I think the reason that you're changes are affecting edgar and mine is because here you're stylizing all buttons on the page, I think, rather than just the ones you select. Before you commit, try changing this to a class and add that class to your buttons and see if that changes things.